### DISCUSSION TITLE
posts/Comments-Enabled/

### DISCUSSION THREAD

**Discussion by giscus** [Equinox134/equinox134.github.io#3] [General]
# posts/Comments-Enabled/

I got comments working

https://equinox134.github.io/posts/Comments-Enabled/

**Comment by Equinox134** [comment]
# Nice.

**Comment by Annyeong-One** [comment]
This equinox guy is the god himself and he is better at geometry than yclock :blobaww:

**Comment by Annyeong-One** [comment]
why is the whole thing so dark tho

**Reply by Annyeong-One** [reply]
and how the fk do I delete my comment

**Comment by c2-h1** [comment]
i randomly stumbled upon your blog and it's quite interesting lol
